### PR TITLE
remove dependence on browser specific APIs fetch and FormData

### LINKED ISSUE #55: Enable Node compatibility by removing dependency on browser fetch API

### PR DESCRIPTION
**Pull request by felixwatts**
resolves #55

### CODE REVIEW

File: src/storage.ts [hunk 1]
```diff
@@ -29,12 +31,13 @@ class IpfsStorageProvider {
       this.settings.port
     }/api/v0/add?pin=true&hash=sha2-256`;
 
-    const formData = new FormData();
-    formData.append("path", msg);
-
+    const formDataBoundary = "2758264728364323843263";
```
Comment (pmackay) [inline src/storage.ts:34]: What is `formDataBoundary`? Where does number come from?

**Review by simonewebdesign — APPROVED** [review]
LGTM 👍🏼

File: package.json [hunk 2]
```diff
@@ -29,13 +29,15 @@
     "bignumber.js": "^9.0.0",
     "canonicalize": "^1.0.1",
     "ethers": "^5.0.19",
+    "isomorphic-fetch": "^3.0.0",
```
Comment (pmackay) [inline package.json:32]: Would this need using for all uses of `fetch` now? Could be good to add a design note in README?

**Review by pmackay — APPROVED** [review]

File: src/storage.ts (hunk 1 above)
Comment (felixwatts) [reply src/storage.ts:34]: Its a string thats unlikely to occur within the data being posted. I just mashed the numpad a bit

File: package.json (hunk 2 above)
Comment (felixwatts) [reply package.json:32]: Done

**Pull request merged**